.cirrus.yml

```diff
@@ -1,18 +1,36 @@
-container:
-  image: gcr.io/flutter-cirrus/build-flutter-image:latest
+# CIRRUS CONFIGURATION FILE
 
+env:
+  # We change Flutter's directory to include a space in its name (see
+  # $CIRRUS_WORKING_DIR) so that we constantly test path names with
+  # spaces in them. The FLUTTER_SDK_PATH_WITH_SPACE variable must
+  # therefore have a space in it.
+  FLUTTER_SDK_PATH_WITH_SPACE: "flutter sdk"
+  GCLOUD_SERVICE_ACCOUNT_KEY: ENCRYPTED[f12abe60f5045d619ef4c79b83dd1e0722a0b0b13dbea95fbe334e2db7fffbcd841a5a92da8824848b539a19afe0c9fb]
+  BOT: "true"
+
+clean_commit_message_script:
```

Comment:
This seems won't take affect to any of the tasks. You can simply move overriding `CIRRUS_CHANGE_MESSAGE` and  `CIRRUS_COMMIT_MESSAGE` to the `env` above.

Reply:
ah yeah that's simpler